request-server_test.go

```diff
@@ -590,6 +604,17 @@ func TestRequestSymlink(t *testing.T) {
 	require.NoError(t, err)
 	assert.Equal(t, []byte("hello"), content)
 
+	fi, err = r.lfetch("/link-to-non-existent-file")
+	require.NoError(t, err)
+	assert.True(t, fi.Mode()&os.ModeSymlink == os.ModeSymlink)
```

Comment:
When using `assert.True` it can be very useful to put in some sort of explaining communication about what this is actually testing, otherwise the test just fails with `expected: true \n got: false`  which is pretty much not helpful at all.

However, I suggest instead, skipping the whole `assert` library† and just write the test clearly:
```
if fi.Mode() & os.ModeSymlink != os.ModeSymlink {
  t.Error("expected mode to be a symlink, but was: ", fi.Mode())
}
```

†: I recommend skiping these assert libraries entirely and always, but we’re already using it heavily here. There is a reason why Golang doesn’t have a standard library assert package, and it’s because it leads people to be careless about their tests, and obfuscates how the code should be used. Namely, test code should be as close to example code as possible. (Bonus: if writing real production-quality code in your tests is cumbersome, it’s a signal to fix your API, not wallpaper over the burden of writing the tests.

Reply:
this was just a copy of the symlink test from the same testcase. I will rework this one and the others aswell